packages/quantic/build-static-resources.js

```diff
@@ -144,7 +144,6 @@ async function main() {
   console.info('Bueno copied.');
 
   await copyLibrary(LIBRARY_CONFIG.headless);
-  await fs.rm(TEMP_DIR, {recursive: true});
```

Comment:
Handled with `--delete-dir-on-start` on babel. (see packages/quantic/package.json#L47)
Changing this allow better caching (incl. cache busting when relevant)